### ISSUE TITLE
History can only be enabled for logged in users

### ISSUE THREAD

**Issue by Aidan223** [Docile-Alligator/Infinity-For-Reddit#1169]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.3.3

### Where did you get the app from

Other

### Android version

Android 13

### Device model

Pixel 7 pro

### First occurred

Today

### Steps to reproduce

1. Go to anonymous mode
2. Go to settings > Post History
3. Shows "Only for Logged-In users"

### Expected behaviour

1. Go to anonymous mode
2. Go to settings > Post History
3. Shows Settings for saving history

### Current behaviour

1. Go to anonymous mode
2. Go to settings > Post History
3. Shows "Only for Logged-In users"

**Comment by Docile-Alligator** [comment]
Yes. It's because I don't want to save the history in anonymous mode.

**Comment by Aidan223** [comment]
You are able to access the History page in anonymous mode, although there is no post showing.

**Comment by Docile-Alligator** [comment]
Oh I see. Thank you for reporting this. I will probably enable post history in anonymous mode.

**Comment by DeathPhantom77** [comment]
Visibility to change Post History is allowed if not logged in or in anonymous mode.
https://github.com/Docile-Alligator/Infinity-For-Reddit/pull/1282
